examples/custom_search_method/asha_search_method/remote_search_runner/run_experiment.py

```diff
@@ -60,15 +60,14 @@
 
         info = det.get_cluster_info()
         assert info is not None
-        args = AttrDict(info.trial.hparams)
 
         # Instantiate your implementation of SearchMethod
         search_method = ASHASearchMethod(
```

Comment:
what do you think about moving the `sample_params` callable to the `run_experiment.py` file for both local and remote search runner?  i think it'll make it clearer that we are sampling from a search space and passing hps to trials and that sampled hps need to correspond to fields within the model_def config.

Reply:
I think that's a great idea! Also, putting `sample_params` at the beginning of each `run_experiment.py` will draw attention to it, which is good. Making the change now!